pages/common/testssl.md

```diff
@@ -0,0 +1,24 @@
+# testssl
+
+> Check SSL/TLS protocols and ciphers supported by a server.
+> More information: <https://testssl.sh/>.
```

Comment:
```suggestion
> More information: <https://testssl.sh>.
```

Reply:
Is that a rule? I see both ways used in existing pages.

Technically, https://testssl.sh/ is the real URL.

Reply:
You're right, the browser hides the `/` :roll_eyes: